pkgs/tools/misc/spigot/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, buildPackages
+, fetchurl
+, gmp
+}:
+
+stdenv.mkDerivation rec {
+  name = "spigot-20200101";
```

Comment:
```suggestion
  pname = "spigot-20200101";
  version = "20200101";
```

https://github.com/NixOS/rfcs/pull/35

Reply:
```suggestion
  version = "20200101";
  name = "spigot-${version}";
```
Maybe even simpler to bump?

Reply:
`pname` is explicitly supposed to be the non-version part of the name. `name` is then derived as `name = "${pname}-${version}"`.